resources/views/user/profile/show.blade.php

```diff
@@ -322,17 +322,15 @@ class="user-search__avatar"
                             @empty
                                 <tr>
                                     <td
-                                        colspan="{{ \config('announce.connectable_check') === true ? 7 : 6 }}"
+                                        colspan="{{ \config('announce.connectable_check') === true ? 8 : 7 }}"
                                     >
                                         No Clients
                                     </td>
                                 </tr>
                             @endforelse
-                        </tbody>
-                        <tfoot>
                             <tr>
                                 <td
-                                    colspan="{{ 5 + (int) auth()->user()->group->is_modo + (int) config('announce.connectable_check') }}"
+                                    colspan="{{ 7 + (int) auth()->user()->group->is_modo + (int) config('announce.connectable_check') }}"
```

Comment:
```suggestion
                                    colspan="{{ 7 + (int) config('announce.connectable_check') }}"
```

If I'm not mistaken, we have one checks too many here.

Reply:
Whoops, you're right, wasn't sure what the moderator check was for and didn't touch it.